apps/charterafrica/src/payload/utils/colors.js

```diff
@@ -0,0 +1,129 @@
+export const validateHexColor = (value = "") => {
```

Comment:
1. Pass value through the original validate, and then through custom one (or vice versa). See [docs](https://payloadcms.com/docs/fields/overview#validation).
2. Use `t` to enable translation for error message (also in the doc above).